dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -208,6 +209,16 @@ export const EuropeLandingModal = ({ edition }: Props) => {
 				});
 			}
 		}
+	}, [editionCookie, modalType]);
```

Comment:
Assuming we needed to add these as we read these values in the callback, even though they are immutable?

Reply:
Yep - eslint gets very angry otherwise!